sketch_rnn.py

```diff
@@ -4,13 +4,20 @@
 	try:
 		import matplotlib
 	except:
-		subprocess.check_call(['sudo', 'dnf', 'install', 'python-matplotlib'])
+		if sys.version_info.major==3:
```

Comment:
Why are your changes specific to Fedora? I use Ubuntu and many folks many other linux distros. I feel users must install the necessary packages in their OS before running this code. I want the code to be platform independent.